src/test/java/conversion/ConverterTest.java

```diff
@@ -0,0 +1,55 @@
+package conversion;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import domain.Car;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class ConverterTest {
+
+    @Test
+    @DisplayName("자동차 이름 변환 작업 테스트 (1): String -> List<Car>")
+    void toCars() {
+        // given
+        String rawCarNames = "aa,bb,cc";
+
+        // when
+        List<Car> carsActual = Converter.toCars(rawCarNames);
+
+        // then
+        assertEquals("aa", carsActual.get(0).getName());
```

Comment:
이렇게도 써볼 수 있어요. 수정할 필요는 없습니다!
```suggestion
        assertThat(carsActual.get(0).getName()).isEqualTo("aa");
```
assertEquals() 와 assertThat().isEqualTo(), assertJ 라이브러리에 대해 한 번 알아보시고 사용해보시는 것도 추천해요!
assertJ 는 특히나 쉽고 나름 유쾌한 공식문서를 가지고 있어서, 처음으로 공식문서를 읽어보기 좋은 입문용 공식문서입니다 :)

Reply:
우테코 이전부터 assertJ를 사용했던 경험이 있어서
연습을 위해 JUnit을 한 번 사용해 보았습니다!
양쪽 다 더 공부해 보겠습니다!

Reply:
연습을 위해 해보는 것 좋네요!
이왕이면 미션하면서 해볼 수 있는거 다 해봐야죠~ 👍